src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -31,25 +32,31 @@ const Body = BaseForm.extend(Object.assign(
     addView() {
       this.add(createButton({
         className: 'button button-wide button-primary send-push link-button-disabled',
-        title: loc('oie.okta_verify.push.sent', 'login'),
+        title: this.isOV() ? loc('oie.okta_verify.push.sent', 'login') : loc('oie.custom_app.push.sent', 'login'),
         click: (e) => {
           e.preventDefault();
         }
       }));
       this.add(
-        `<span class='accessibility-text' role='alert'>${loc('oie.okta_verify.push.sent', 'login')}</span>`,
+        `<span class='accessibility-text' role='alert'>
+        ${this.isOV() ? loc('oie.okta_verify.push.sent', 'login') : loc('oie.custom_app.push.sent', 'login')}</span>`,
       );
     },
 
     postRender() {
+      const className = this.isOV() ?
```

Comment:
nit: can this be in `initialize`?

Reply:
Classname is concatenated and appended in the constructor so wont work.